Libraries/react-native/react-native-implementation.js

```diff
@@ -172,9 +172,6 @@ module.exports = {
   get Alert() {
     return require('Alert');
   },
-  get AlertIOS() {
```

Comment:
This is used in other places. Can you search for `AlertIOS` in the repository and remove uses of this module and update tests to use `Alert` instead?

Here, it might make sense to actually throw and tell users to use `Alert` instead.

Reply:
Okay, I'll do that.